src/main/resources/assets/minecolonies/lang/en_us.json

```diff
@@ -1116,9 +1114,9 @@
   "com.minecolonies.coremod.gui.citizen.stacks": "Blocks to Request:",
   "com.minecolonies.coremod.gui.workerhuts.request": "Request",
 
-  "com.minecolonies.coremod.gui.workerhuts.blacksmith": "Blacksmith",
+  "com.minecolonies.coremod.gui.workerhuts.blacksmith": "Blacksmith's Hut",
   "com.minecolonies.coremod.gui.workerhuts.stonemason": "Stonemason",
-  "com.minecolonies.coremod.gui.workerhuts.smelter": "Metal Smeltery",
+  "com.minecolonies.coremod.gui.workerhuts.smelter": "Smeltery",
```

Comment:
Forge makes me think of the Blacksmith's Hut though.

Reply:
Fair enough. They often came in village blacksmith shops anyway. If anything the concept of a stone smeltery doesn't really exist in history AFAIK. So I guess this will do.